tools/app/.env.test

```diff
@@ -1 +1 @@
-npm_config_project_path="../data-handler/test/test-data/valid/decision-records"
\ No newline at end of file
+npm_config_project_path="../../../cyberismo-bat"
```

Comment:
This breaks API tests that have relied on the previously used data-handler test project. It's an annoying limitation that we can only have one project_path in a given env...can't think of a good workaround right now!

Reply:
I tried fixing this by using a separate NODE_ENV for e2e tests (".env.test-e2e") which worked locally BUT caused the following warning:

You are using a non-standard "NODE_ENV" value in your environment. This creates inconsistencies in the project and is strongly advised against. Read more: https://nextjs.org/docs/messages/non-standard-node-env

...so that is not a clean solution. I'll think about this more.